vince-fugnitto/local-history-ext
# Add tree-view view-container

**Description**
- List all revisions for the active editor in a tree-view view-container
- Click on a revision in tree-view to open it in diff mode
- Delete a revision in the tree-view using the `Trash` icon
- Refresh entries in the tree-view by clicking the `Refresh` button or changing active editor
- Display message in the tree-view when no revision is found
- Update all variable name for `provider` to `manager` since it has type LocalHistoryManager

**How to test**
1. When there is no active editor/no revision is found, you should see a mesage in tree-view: `No local history found`
2. Open a file with local history, hover over a tree node to see tooltip text `Open '123.ts'`
3. Click on a revision in tree-view to open it in diff mode
4. Delete a revision in the tree-view using the `Trash` icon
5. When all revisions for an active editor are deleted, you should see a mesage in tree-view: `No local history found`


Signed-off-by: Kaiyue Pan <kaiyue.pan@ericsson.com>
